sui/src/sui-move.rs

```diff
@@ -12,7 +12,11 @@ use structopt::StructOpt;
 pub enum MoveCommands {
     /// Build and verify Move project
     #[structopt(name = "build")]
-    Build,
+    Build {
+        /// Whether we are printing in hex.
+        #[structopt(long)]
+        hex: bool,
```

Comment:
I recommend renaming this to make it more explicit. Something like dump_bytecode_as_hex